app/Helpers/database/player-history.php

```diff
@@ -102,6 +103,7 @@ function getAchievementsEarnedOnDay(int $unixTimestamp, User $user): array
 
 function getAwardedList(
     User $user,
+    bool $excludeEvents = true,
```

Comment:
new optional parameters should be added at the end of the list to not break any other existing calls that rely on the existing optional parameters having specific indices.

it looks like this function is only called twice, and the other case isn't passing any optional parameters, so this is probably fine.